File: New-Caddus/Features-Solo/Cesar/Cesar.py
#CESAR SYSTEM
       print("olá, bem vindo ao Joseph")
       print("perfilador Caddus.\n")
       print("Opções:")
       print("1-adicionar pessoas")
       print("2-Ver uma pessoa")
       perfopt1 = input("opção: ")
       if (perfopt1 == "1"):
          os.system("clear") or None
          print("qual pessoa você deseja Salvar?")
          pessoaPerf == input("nome: ")
          pessoaSPerf == input("sobrenome: ")
          print("digite as características que você conhece")
          caracPerf == input("caracteristicas: ")
          PESSOAWPERF == open(f"/Perfis/{pessoaPerf}_{pessoaSPerf}.txt", "w")
          PESSOAWPERF.write(f"\nNome: {pessoaPerf}\nDados: {caracPerf}")
          exit()
       if (perfopt1 == "2"):
          pessoaPerf == input("nome: ")
          pessoaSPerf == input("sobrenome: ")
          PESSOARPERF == open(f"/Perfis/{pessoaPerf}_{pessoaSPerf}.txt", "r")
          PESSOARPERF.read()
          nothing == input("Digite enter para sair")
          exit()
      else:
          print("Erro: Escolha um número correto")
          time.sleep(2)
          exit()

File: New-Caddus/New-Caddus/CaddusX/Cesar.rb
puts 'olá, bem vindo ao Cesar'
puts "perfilador Caddus.\n"
puts "Opções:\n1-adicionar pessoas\n2-Ver uma pessoa"
print 'Opção: '
perfopt1 = gets.chomp
if perfopt1 == "1"
   system("clear")
   puts 'qual pessoa você deseja Salvar?'
   print 'nome: '
   pessoaPerf = gets.chomp
   print 'sobrenome: '
   pessoaSPerf = gets.chomp
   puts 'digite as características que você conhece'
   print 'caracteristicas: '
   caracPerf = gets.chomp
   PESSOAWPERF = File.open("/Perfis/#{pessoaPerf}_#{pessoaSPerf}.txt", "w")
   PESSOAWPERF.write("\nNome: #{pessoaPerf}\nDados: #{caracPerf}")
   exit
elsif perfopt1 == "2"
   print 'nome: '
   pessoaPerf = gets.chomp
   print 'sobrenome: '
   pessoaSPerf = gets.chomp
   PESSOARPERF = File.open("/Perfis/#{pessoaPerf}_#{pessoaSPerf}.txt", "r")
   PESSOARPERF.read()
   print 'Digite enter para sair'
   nothing = gets.chomp
   exit
else
   puts("Erro: Escolha um número correto")
   sleep 2
   exit
end